packages/text/src/TextStyle.ts

```diff
@@ -57,6 +89,38 @@ const genericFontFamilies = [
  */
 export class TextStyle
 {
+    public styleID: number;
+    
+    protected _align: ITextStyle['align'];
```

Comment:
Don't use lookup types.

Reply:
I can get rid of those. I agree that it's not really nice for classes to reference each other, but given that TextStyle implements ITextStyle I figured it would make sense to specify the types in only one place. But I'll change those to not use lookup, I'll just need to declare a bunch of extra types for the interface so I don't have to type for example `'normal'|'bold'|'bolder'|'lighter'|'100'|'200'|'300'|'400'|'500'|'600'|'700'|'800'|'900'` more than once.

Reply:
Create separate types for things like this maybe? TextStyleAlign, TextStyleWeight, etc

Reply:
Yep that's what I did. I didn't have the TextStyle prefix but I just added that to all of them - I think that makes it a bit clearer. I'll push in a bit.